src/widget/header/ui/header.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react'
+import {
+    Flex,
+    Group,
+    Image,
+    Select,
+    Switch,
+    rem,
+    useMantineTheme,
+} from '@mantine/core'
+
+import { IconSun, IconMoonStars, IconWorld } from '@tabler/icons-react'
+export const Header: FC = () => {
+    const theme = useMantineTheme()
+
+    const sunIcon = (
```

Comment:
put all the icons in one object and pick them up by key

`const icons = {
        sun: (
            <IconSun
                style={{ width: rem(16), height: rem(16) }}
                stroke={2.5}
                color={theme.colors.yellow[4]}
            />
        ),
        moon: (
            <IconMoonStars
                style={{ width: rem(16), height: rem(16) }}
                stroke={2.5}
                color={theme.colors.blue[6]}
            />
        ),
        world: (
            <IconWorld
                style={{ width: rem(18), height: rem(18) }}
                stroke={1.5}
            />
        ),
    }`
    
    like this